### PR TITLE
Dashboard alterations

### LINKED ISSUE #122: Dashboard - colours
Change background colour and navigation bar colour to match colours used in the presentation

As an engineer I need to create a theme across all outputs and match colours where possible

### PR DESCRIPTION
**Pull request by alina-101**
Added Apollo logo to home page
Changed headings and subheadings
Changed background colour of dashboard

closes #122

### CODE REVIEW

File: dashboard/Apollo.png
Comment (Lasped13) [inline dashboard/Apollo.png:1]: Wow look so good.

File: dashboard/.streamlit/config.toml
Comment (Lasped13) [inline dashboard/.streamlit/config.toml:1]: Looks fine to me.

File: dashboard/pages/home.py [hunk 1]
```diff
@@ -12,9 +12,15 @@ def show_home():
...
-    st.write("""Here you'll find insights into Bandcamp sales
-             and you can also subscribe to receive email notifications!""")
+    col = st.columns(2)
+    with col[0]:
+        st.image('Apollo.png', use_column_width="auto")
+    with col[1]:
+        st.header("Welcome to Apollo!")
+        st.subheader("The Bandcamp Sales Tracker")
+        st.write(
+            "Here you'll find insights into Bandcamp sales and you can also subscribe to receive email notifications!")
```
Comment (Lasped13) [inline dashboard/pages/home.py:22]: Is this too long for pylint? Can't be more than 100 characters.

File: dashboard/.streamlit/config.toml [hunk 2]
```diff
@@ -4,6 +4,8 @@ gatherUsageStats = false
 [theme]
 base = "light"
 # Bandcamp blue
+backgroundColor = "#fff7e2"
+secondaryBackgroundColor = 'fff0c9'
 primaryColor = "#3c9aaa"
```
Comment (e-lemma) [inline dashboard/.streamlit/config.toml:9]:
```suggestion
backgroundColor = "#fff7e2"
secondaryBackgroundColor = 'fff0c9'
# Bandcamp blue
primaryColor = "#3c9aaa"
```

Shouldn't it be like this?

File: dashboard/Apollo.svg
Comment (e-lemma) [inline dashboard/Apollo.svg:1]: Why is there both a SVG and PNG of the same image?

File: dashboard/charts.py [hunk 3]
```diff
@@ -85,19 +85,18 @@ def create_choropleth_map(locations):
 
 
 @st.cache_data
-def get_albums_sales_line_graph(chosen_album) -> alt.Chart:
+def get_artist_sales_line_graph(chosen_artist) -> alt.Chart:
     """Returns a line graph of sales over time for a chosen album."""
```
Comment (e-lemma) [inline dashboard/charts.py:89]: I think this docstring needs to be updated, judging by the new function name

File: dashboard/charts.py [hunk 4]
```diff
@@ -85,19 +85,18 @@ def create_choropleth_map(locations):
...
         x=alt.X("timestamp:T"),
         y=alt.Y("sales:Q"),
-        color="name:N"
     ).interactive()
 
 
 @st.cache_data
-def get_tag_sales_line_graph(chosen_tag) -> alt.Chart:
+def get_sales_line_graph(chosen_data) -> alt.Chart:
     """Returns a line graph of sales over time for a chosen tag"""
```
Comment (e-lemma) [inline dashboard/charts.py:98]: Does this docstring also need to be changed?

File: dashboard/database.py [hunk 5]
```diff
@@ -230,27 +230,28 @@ def get_all_album_titles(_conn: Connection):
 
 
 @st.cache_data(ttl="1hr")
-def get_album_sales_by_album(_conn: Connection, album_name: str):
+def get_track_sales_by_artist(_conn: Connection, artist: str):
     """Returns all album info for a given album."""
```
Comment (e-lemma) [inline dashboard/database.py:234]: Docstring needs updating!

File: dashboard/Apollo.svg
Comment (alina-101) [reply dashboard/Apollo.svg:1]: Both are being used. One is used for the image in the navigation bar. And the other on the home page

File: dashboard/charts.py (hunk 3 above)
Comment (alina-101) [reply dashboard/charts.py:89]: You are right. Good spot thank you

File: dashboard/charts.py (hunk 4 above)
Comment (alina-101) [reply dashboard/charts.py:98]: Yes just changed it

File: dashboard/pages/home.py (hunk 1 above)
Comment (alina-101) [reply dashboard/pages/home.py]: You're right. I've improved the pylint score by changing this

File: dashboard/Apollo.svg
Comment (e-lemma) [reply dashboard/Apollo.svg:1]: I see...

**Review by e-lemma — APPROVED** [review]
All feedback and comments have been addressed, OK to merge!

**Pull request merged**